SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -641,96 +756,13 @@ void CFlowOutput::SetAnalyzeSurface(const CSolver* const*solver, const CGeometry
   }
 
   /*--- Set values on markers ---*/
-  for (iMarker_Analyze = 0; iMarker_Analyze < nMarker_Analyze; iMarker_Analyze++) {
+  for (unsigned short iMarker_Analyze = 0; iMarker_Analyze < nMarker_Analyze; iMarker_Analyze++) {
     su2double SpeciesVariance = Surface_SpeciesVariance_Total[iMarker_Analyze];
     SetHistoryOutputPerSurfaceValue("SURFACE_SPECIES_VARIANCE", SpeciesVariance, iMarker_Analyze);
     Tot_Surface_SpeciesVariance += SpeciesVariance;
     // Set value into config. Necessary to access as an OF.
```

Comment:
It's actually not needed, you can get it from the output, check if you can make the config const please.

Reply:
I am currently following the template which is used to set the OF for the DA. I actually used the Config to set the variance OF for this https://github.com/su2code/SU2/pull/1388#issuecomment-982771284 below. So I keep that in mind for the adjoint PR that will come next

Reply:
So I would postpone this thought to the species adjoint PR